lib/mcinstall/index.js

```diff
@@ -0,0 +1,64 @@
+import {BaseServicePlist} from '../base-service';
+import {fs, plist} from '@appium/support';
+
+
+const MC_INSTALL_SERVICE_NAME = 'com.apple.mobile.MCInstall';
+const ACKNOWLEDGED = 'Acknowledged';
+
+class MCInstallProxyService extends BaseServicePlist {
+  constructor (socketClient) {
+    super(socketClient);
+  }
+
+  checkACK (res, name) {
```

Comment:
I assume this could be a separate function since it does not use any class-specific calls